bound/kt/src/test/kotlin/web5/sdk/vc/VerifiableCredentialTest.kt

```diff
@@ -572,6 +572,22 @@ class VerifiableCredentialTest {
 
             assertEquals("claim mismatch: expiration_date", exception.msg)
         }
+    }
+
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    inner class DataModelValidation {
```

Comment:
we want to keep the unit test suites 1:1 to the APID features, because we're testing the public api surface. if we want to add a publicly facing feature for data model validation then we can do this, otherwise it must be embedded in the `from_vc_jwt` like it was